client/settings/advanced-settings/multi-currency-toggle.js

```diff
@@ -17,14 +16,6 @@ const MultiCurrencyToggle = () => {
 		updateIsMultiCurrencyEnabled,
 	] = useMultiCurrency();
 
-	const headingRef = useRef( null );
```

Comment:
`headingRef` is assigned here - but it's not assigned to a DOM element. Unlike it was done here: https://github.com/Automattic/woocommerce-payments/blob/8f24a5d98c6938d88f537fcdfe5435772bb30379/client/settings/advanced-settings/debug-mode.js#L26